resolvers.js

```diff
@@ -255,6 +255,26 @@ const Resolvers = {
 
     uploadDateTime(ds) {
       return ds._source.ds_upload_dt;
+    },
+
+    fdrCounts(ds, {inpFdrLvls}) {
+      if(ds._source.ds_status == 'FINISHED') {
+        let inpAllLvlCounts = ds._source.annotation_counts[0].counts;
+        let outfdrLvls = [], outFdrCounts = [];
+        inpFdrLvls.forEach(lvl => {
+          for (let lvlCount of inpAllLvlCounts) {
```

Comment:
I'd use `inpAllLvlCounts.find()` instead of iteration through the whole array. It'd do the same thing, but read better